requirements.txt

```diff
@@ -6,7 +6,7 @@ sphinx-autobuild
 sphinxcontrib-plantuml
 recommonmark
 sphinx-rtd-theme
-pytest-bdd
+pytest-bdd==3.4.0
```

Comment:
I think there's a `<4.0.0` syntax that ought to work too. Let's see if this works to fix the build errors first!

Reply:
It does! Since https://pypi.org/project/pytest-bdd/4.0.0/#migration-of-your-tests-from-versions-3-x-x suggests that it is v4.0.0 that introduces the problem, presumably if for whatever reason they update 3.x.x then they wouldn't introduce the same breaking change and we would want to pick up whatever fixes it includes.